Integrations/CentralBankIntegrationLib/XmlSerializationObjects/ValCurse.cs

```diff
@@ -13,6 +13,6 @@ public class ValCurs
         [XmlElement("Valute")]
         public List<Valute> Valutes { get; set; }
 
-        public List<string> TopValutesCharCode => new List<string> {  };
+        public List<string> TopValutesCharCode => new List<string> { "EUR", "USD", "GBR", "CNY", "JPY" };
```

Comment:
We need this change because code this integration lib is not correct

Reply:
Change back

Reply:
why is it not correct ? if I change the code, it will always output "No currencies found" in method  CentralBankNewsPublisher.GetTopValutes

Reply:
I didn't find in the code that someone filled this list